[2023-12-19T04:46:20.451+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Automobile_Insurance-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_incident_severity scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2023-12-19T04:46:20.479+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Automobile_Insurance-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_incident_severity scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2023-12-19T04:46:20.485+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-12-19T04:46:20.506+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_fk_col_in_fact_table_incident_severity> on 2023-12-01 00:00:00+00:00
[2023-12-19T04:46:20.522+0000] {standard_task_runner.py:57} INFO - Started process 1273 to run task
[2023-12-19T04:46:20.527+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Automobile_Insurance-v01.insert_fk_into_fact_table', 'insert_fk_col_in_fact_table_incident_severity', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/Automobile Insurance.py', '--cfg-path', '/tmp/tmpaw12gntq']
[2023-12-19T04:46:20.529+0000] {standard_task_runner.py:85} INFO - Job 49: Subtask insert_fk_col_in_fact_table_incident_severity
[2023-12-19T04:46:20.610+0000] {task_command.py:416} INFO - Running <TaskInstance: Automobile_Insurance-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_incident_severity scheduled__2023-12-01T00:00:00+00:00 [running]> on host 07f4bc2fb49b
[2023-12-19T04:46:20.832+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dunnbc22' AIRFLOW_CTX_DAG_ID='Automobile_Insurance-v01.insert_fk_into_fact_table' AIRFLOW_CTX_TASK_ID='insert_fk_col_in_fact_table_incident_severity' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2023-12-19T04:46:20.841+0000] {sql.py:274} INFO - Executing: -- params: table_name, dimension_col

/*
Steps 1-3 were completed in the create_dim_table.sql 
file & associated PostgresOperator
*/


-- Step 4: Add foreign key column to fact table
ALTER TABLE automobile_insurance
ADD COLUMN incident_severity_id INTEGER;

-- Step 4: Update foreign key column to fact table with 
-- appropriate id (that matches with dim table)
UPDATE automobile_insurance
SET incident_severity_id = dimension_table_incident_severity.id
FROM dimension_table_incident_severity
WHERE COALESCE(automobile_insurance.incident_severity::text, '') = COALESCE(dimension_table_incident_severity.incident_severity::text, '');
[2023-12-19T04:46:20.871+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T04:46:20.900+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T04:46:20.924+0000] {sql.py:418} INFO - Running statement: -- params: table_name, dimension_col

/*
Steps 1-3 were completed in the create_dim_table.sql 
file & associated PostgresOperator
*/


-- Step 4: Add foreign key column to fact table
ALTER TABLE automobile_insurance
ADD COLUMN incident_severity_id INTEGER;

-- Step 4: Update foreign key column to fact table with 
-- appropriate id (that matches with dim table)
UPDATE automobile_insurance
SET incident_severity_id = dimension_table_incident_severity.id
FROM dimension_table_incident_severity
WHERE COALESCE(automobile_insurance.incident_severity::text, '') = COALESCE(dimension_table_incident_severity.incident_severity::text, '');, parameters: None
[2023-12-19T04:46:20.978+0000] {sql.py:427} INFO - Rows affected: 1000
[2023-12-19T04:46:21.001+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Automobile_Insurance-v01.insert_fk_into_fact_table, task_id=insert_fk_col_in_fact_table_incident_severity, execution_date=20231201T000000, start_date=20231219T044620, end_date=20231219T044621
[2023-12-19T04:46:21.082+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-19T04:46:21.218+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
